### PR TITLE
Feat: Created Header component

### PR DESCRIPTION
**Pull request by Justin-ml**
# Desription:

## Changes:

- Created Navbar component
- Created Usericon component
- Integrated both component in App component.

### screenshots

Header  Component

### CODE REVIEW

File: src/Components/App/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import NavBar from "../Navbar/NavBar";
+
```
Comment (Hellboyksr2) [inline src/Components/App/index.js]: @Justin-ml Please format the Code Before Commiting.

File: src/Components/Icon/UserIcon.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Icon } from 'semantic-ui-react';
+import usericon from './userIcon.css'
+
+const UserIcon = () => (
+  <div className="user-icon">
+           <Icon.Group size='large'>
```
Comment (Hellboyksr2) [inline src/Components/Icon/UserIcon.js]: @Justin-ml Same as Above

File: src/Components/Icon/UserIcon.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Icon } from 'semantic-ui-react';
+import usericon from './userIcon.css'
```
Comment (Hellboyksr2) [inline src/Components/Icon/UserIcon.js]: @Justin-ml Why do we need `usericon` for CSS imports?

File: src/Components/Navbar/NavBar.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import navBar from "../Navbar/navBar.css";
```
Comment (Hellboyksr2) [inline src/Components/Navbar/NavBar.js]: Same as Above

File: src/Components/Navbar/navBar.css [hunk 5]
```diff
@@ -0,0 +1,21 @@
+*{
```
Comment (Hellboyksr2) [inline src/Components/Navbar/navBar.css]: Format Code Plzzz

File: src/Components/App/index.js (hunk 1 above)
Comment (Justin-ml) [reply src/Components/App/index.js]: Done.

File: src/Components/Icon/UserIcon.js (hunk 2 above)
Comment (Justin-ml) [reply src/Components/Icon/UserIcon.js]: Done.

File: src/Components/Icon/UserIcon.js (hunk 3 above)
Comment (Justin-ml) [reply src/Components/Icon/UserIcon.js]: resolved that part.

File: src/Components/Navbar/NavBar.js (hunk 4 above)
Comment (Justin-ml) [reply src/Components/Navbar/NavBar.js]: done.

File: src/Components/Navbar/navBar.css (hunk 5 above)
Comment (Justin-ml) [reply src/Components/Navbar/navBar.css]: Done.

**Pull request merged**